### DISCUSSION TITLE
You took something great and made it FAR better

### DISCUSSION THREAD

**Discussion by yuckypants** [ngocjohn/vehicle-status-card#4] [General]
One of the best I've ever seen. I liked Ultra-Vehicle-Card, but it wasn't without a few issues. But your card...capitalizes on all of the positives and fixes many of the negatives. 

You also have clear and concise directions, which is fantastic. 

Just wanted to come here and say it!

**Comment by ngocjohn** [comment]
Hi @yuckypants! Thanks for your kind words, in the future I have other improvements ready, for example Tap actions, Tire pressures with car background etc..  🚀 
![custom-tire-bg](https://github.com/user-attachments/assets/760251e0-cf31-47c2-9b4a-30c9b0d41bfc)
